webpack/assets/javascripts/react_app/common/I18n/intl.js

```diff
@@ -0,0 +1,7 @@
+const [htmlElemnt] = document.getElementsByTagName('html');
```

Comment:
For my own understanding: why is this `[htmlElement]` and not `htmlElemnt`?

Reply:
The `getElementsByTagName` returns an array and we want the first item in the array (because there is only one HTML tag).

This feature (destructuring assignment) exits in javascript for a while:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

You can do the same with objects which is the more familiar version of it. (`const { a, b, ...rest } = someObject;`)